src/providers/export/tensorFlowPascalVOC.ts

```diff
@@ -75,7 +75,7 @@ export class TFPascalVOCJsonExportProvider extends ExportProvider<ITFPascalVOCJs
     }
 
     private async exportSingleImage(jpegImagesFolderName: string, element: IAssetMetadata): Promise<void> {
-        const imageFileName = `${jpegImagesFolderName}/${element.asset.name}`;
+        const imageFileName = `${jpegImagesFolderName}/${element.asset.name.replace(/\.[^/.]+$/, ".jpg")}`;
```

Comment:
Agreed - need some more tests.  How can we validate that this happens consistently across providers?

Reply:
@wbreza I agree, but I'm not sure how to do this given the tests we have are already "testing" that assets are exported to "jpg" images. Let's have a discussion offline as to how to refactor them to test the correct thing.

Reply:
Moved fix into HtmlFileReader and added tests.